src/Jackett.Common/Indexers/TorrentHeaven.cs

```diff
@@ -192,50 +193,50 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                 var rows = dom.QuerySelectorAll("table.torrenttable > tbody > tr");
                 foreach (var row in rows.Skip(1))
                 {
-                    var release = new ReleaseInfo();
-                    release.MinimumRatio = 0.8;
-                    release.MinimumSeedTime = 0;
                     var qDetailsLink = row.QuerySelector("a[href^=\"index.php?strWebValue=torrent&strWebAction=details\"]");
-                    release.Title = titleRegexp.Match(qDetailsLink.GetAttribute("onmouseover")).Groups[1].Value;
                     var qCatLink = row.QuerySelector("a[href^=\"index.php?strWebValue=torrent&strWebAction=search&dir=\"]");
                     var qDlLink = row.QuerySelector("a[href^=\"index.php?strWebValue=torrent&strWebAction=download&id=\"]");
                     var qSeeders = row.QuerySelectorAll("td.column1")[3];
                     var qLeechers = row.QuerySelectorAll("td.column2")[3];
                     var qDateStr = row.QuerySelector("font:has(a)");
                     var qSize = row.QuerySelector("td.column2[align=center]");
                     var catStr = qCatLink.GetAttribute("href").Split('=')[3].Split('#')[0];
-                    release.Category = MapTrackerCatToNewznab(catStr);
-                    release.Link = new Uri(SiteLink + qDlLink.GetAttribute("href"));
-                    release.Comments = new Uri(SiteLink + qDetailsLink.GetAttribute("href"));
-                    release.Guid = release.Link;
+                    var link = new Uri(SiteLink + qDlLink.GetAttribute("href"));
                     var sizeStr = qSize.TextContent;
-                    release.Size = ReleaseInfo.GetBytes(sizeStr);
-                    release.Seeders = ParseUtil.CoerceInt(qSeeders.TextContent);
-                    release.Peers = ParseUtil.CoerceInt(qLeechers.TextContent) + release.Seeders;
-                    var dateStr = qDateStr.TextContent.Trim();
-                    var dateStrParts = dateStr.Split();
-                    var dateGerman = dateStrParts[0] switch
-                    {
-                        "Heute" => (DateTime.SpecifyKind(DateTime.UtcNow.Date, DateTimeKind.Unspecified) + TimeSpan.Parse(dateStrParts[1])),
-                        "Gestern" => (DateTime.SpecifyKind(DateTime.UtcNow.Date, DateTimeKind.Unspecified) + TimeSpan.Parse(dateStrParts[1])
-                                      - TimeSpan.FromDays(1)),
-                        _ => DateTime.SpecifyKind(                            DateTime.ParseExact(
-                                dateStrParts[0] + dateStrParts[1], "dd.MM.yyyyHH:mm", CultureInfo.InvariantCulture),
-                            DateTimeKind.Unspecified)
-                    };
+                    var releaseSeeders = ParseUtil.CoerceInt(qSeeders.TextContent);
```

Comment:
I tried to reduce diff size because I knew how large this kind of refactor was going to be. So this was affected by the order I was making changes to the code, specifically going ahead of your initial review to pull out other values inside the initializer.